scripts/generate_test_code.py

```diff
@@ -0,0 +1,1277 @@
+#!/usr/bin/env python3
```

Comment:
The first commit in the history is "Rename generate_code.py -> generate_test_code.py". It's a bit of an annoying edge case, but it would be nice to get the history from before the renaming too.

Reply:
This seems to be non-trivial, but there does seem to be a [tool](https://github.com/rainlabs-eu/git-relevant-history) that may be able to do it. I'll give it a try.